import-scripts/filter-clinical-arg-functions.sh

```diff
@@ -87,24 +87,48 @@ function find_clinical_attributes_to_filter_arg() {
 
 function filter_clinical_attribute_columns() {
     # Path to clinical file taken as an argument
-    clinical_attribute_filepath="$1"
+    clinical_filepath="$1"
 
     # Attributes to deliver in the clinical file
     attributes_to_deliver="$2"
 
-    # Path to output clinical file taken as an argument
-    output_filepath="$3"
+    # Temporary output file, removed at end of function
+    tmp_processing_file=$(mktemp -q)
```

Comment:
minor -- but wonder if there's a way to get this tmpfile name in the case proccess fails and we want to check what the output was

Reply:
just added a logging line to do this